hassio/src/addon-store/hassio-addon-store.ts

```diff
@@ -92,28 +95,52 @@ class HassioAddonStore extends LitElement {
         .tabs=${supervisorTabs}
       >
         <span slot="header">Add-on store</span>
-        <ha-icon-button
-          icon="hassio:reload"
+        <paper-menu-button
+          close-on-activate
+          no-animations
+          horizontal-align="right"
+          horizontal-offset="-5"
           slot="toolbar-icon"
-          aria-label="Reload add-ons"
-          @click=${this.refreshData}
-        ></ha-icon-button>
-
+        >
+          <ha-icon
+            icon="hassio:dots-vertical"
+            slot="dropdown-trigger"
+            alt="menu"
+          ></ha-icon>
+          <paper-listbox slot="dropdown-content" role="listbox">
+            <paper-item @tap=${this._manageRepositories}>
+              Repositories
+            </paper-item>
+            <paper-item @tap=${this.refreshData}>
+              Reload
+            </paper-item>
+          </paper-listbox>
+        </paper-menu-button>
         ${repos.length === 0
           ? html`<loading-screen></loading-screen>`
           : html`
-              <hassio-repositories-editor
-                .hass=${this.hass}
-                .repos=${this._repos!}
-              ></hassio-repositories-editor>
-
-              <hassio-search-input
-                .filter=${this._filter}
-                @value-changed=${this._filterChanged}
-              ></hassio-search-input>
+              <div class="search">
+                <search-input
+                  no-label-float
+                  no-underline
+                  .filter=${this._filter}
+                  @value-changed=${this._filterChanged}
+                ></search-input>
+              </div>
 
               ${repos}
             `}
+        ${!this.hass.userData?.showAdvanced
+          ? html`
+              <div class="advanced">
+                Missing add-ons? Enable advanced mode on
+                <div class="advanced-link" @click=${this._openProfile}>
```

Comment:
Make this an `<a>`

Reply:
The store runs in an iframe, an a would cause it to load the entire site (dual sidebars)

Reply:
maybe not iframe I do not quite remember, but something.
It was an `<a>` initially, but I had to change that.

Reply:
you can set the target to `_top` or `_parent`

Reply:
Thanks! I'll try that

Reply:
It worked 🎉